go-guru.el

```diff
@@ -138,6 +138,11 @@
     "---"
     ["Set pointer analysis scope..." go-guru-set-scope t]))
 
+(defun go-guru--read-scope ()
+  "Read a godoc query from the minibuffer."
+  (completing-read-multiple "guru-scope; "
+                            (go-packages) nil nil nil 'go-godoc-history))
```

Comment:
Wrong docstring, and using the wrong history (should be `go-guru--scope-history`). Also, there's a semicolon instead of a colon in the prompt, and the prompt should indicate that multiple, comma-separated elements are permitted. `guru-scope (comma-separated):` might be good enough, unless you can think of something better.